net/socks5/server.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package socks5 is an auth-less SOCKS5 server implementation
```

Comment:
drop "auth-less". that's not a defining feature of this. it's just the current state of things.